File: classes/parte-11-proyecto-integrador-poliglota/173-pruebas-end-to-end-del-sistema-completo/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	f := strings.Fields(line)
	a, _ := strconv.Atoi(f[0])
	b, _ := strconv.Atoi(f[1])
	e, _ := strconv.Atoi(f[2])
	res := "falla"
	if a+b == e {
		res = "pasa"
	}
	fmt.Printf("e2e=%s\n", res)
}

File: classes/parte-11-proyecto-integrador-poliglota/173-pruebas-end-to-end-del-sistema-completo/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long a, b, e;
    if (scanf("%ld %ld %ld", &a, &b, &e) != 3) return 1;
    printf("e2e=%s\n", a + b == e ? "pasa" : "falla");
    return 0;
}
